<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_userID">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>user : test</string>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>721</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="chat_room_list_layout">
     <item>
      <widget class="QListWidget" name="room_list"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="btnMakeroom">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>40</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>방 만들기</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_saero">
    <property name="geometry">
     <rect>
      <x>654</x>
      <y>112</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>새로고침</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <widget class="QMenu" name="menu_2">
     <property name="title">
      <string>설정</string>
     </property>
     <addaction name="actiontest"/>
     <addaction name="actiontest_2"/>
    </widget>
    <addaction name="action_logout"/>
    <addaction name="menu_2"/>
    <addaction name="action_exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_logout">
   <property name="text">
    <string>로그아웃</string>
   </property>
  </action>
  <action name="actiontest">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="actiontest_2">
   <property name="text">
    <string>test</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="text">
    <string>종료</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
